test/config/TestConfigSerialization.cpp

```diff
@@ -138,45 +123,14 @@ INSTANTIATE_TEST_CASE_P(
             }
         ]
     }
-})",
-        R"({
```

Comment:
I have removed this tests since it looks redundant.

Reply:
I think these tests are roundtrip tests to verify the string used as input to step 1 is the same as the string returned as output in step 2.
1. read config from a string into json object using `LoadFromJson`
2. return string from in-memory json object using `SerializeToObject`

Reply:
Okay. I have separated the older test into two different tests.